pages/jobs/index.js

```diff
@@ -33,21 +45,36 @@ export default function JobsIndexPage() {
              Job Postings
             </h2>
             <p className="mt-3 max-w-2xl mx-auto text-xl leading-7 text-gray-500 sm:mt-4">
-              Want to work on a great open-source project with a lovely team and a vibrant community? Come work with us!
+              Want to work on a great open-source project with a lovely team and a vibrant community? Browse through available job offers!
+            </p>
+            <p className="max-w-2xl mx-auto text-md leading-7 text-gray-400">
+            <span role="img">💡</span> Want to post a job offer? All jobs must be AsyncAPI-related. <a className="ml-1 text-primary-500 hover:text-primary-400" href={`https://github.com/asyncapi/website/new/master/pages/jobs?value=${jobPostUrl}`} target="_blank">Get started now!</a>
```

Comment:
What about this?

```suggestion
            <span role="img">💡</span> Want to post a job offer? <a className="ml-1 text-primary-500 hover:text-primary-400" href={`https://github.com/asyncapi/website/new/master/pages/jobs?value=${jobPostUrl}`} target="_blank">Post it now!</a>
```

The disclaimer doesn't need to be here. It can be [here](https://github.com/asyncapi/website/new/master/pages/jobs?value=---%0Atitle%3A%20%27Job%20Title%27%0Adate%3A%20DD%2FMM%2FYYYY%20(current%20date)%0Acategory%3A%20job%20category%0AclosingOn%3A%20MM%2FDD%2FYYYY%0A---%0A%23%23%20About%20the%20team%0ALet%20potential%20employees%20knows%20what%20it%27ll%20feel%20like%20to%20joining%20your%20team%20in%20this%20section%0A%0A%23%23%20TL%3BDR%0AEnter%20list%20of%20perks%20for%20potential%20employees%20in%20this%20section.%0AE.g.%0A*%20%3Amuscle%3A%20Growing%20team%0A*%20%3Ahouse_with_garden%3A%20Fully%20remote%20job%0A*%20%3Amoney_mouth_face%3A%20Great%20salary%20and%20compensation%20package%0A*%20%3Amountain_snow%3A%20Unlimited%20paid%20time%20off%0A%0A%23%23%20About%20the%20job%0ALet%20the%20potential%20employee%20knows%20about%20the%20job%20in%20this%20section%0A%0A%23%23%20About%20youY%0AThe%20Potential%20employees%20would%20love%20to%20know%20more%20about%20you.%20Tell%20them%20more%20about%20you%20in%20this%20section%0A%0A%23%23%20Pay%20and%20benefits%0ALet%20the%20potential%20employees%20knows%20what%20comes%20with%20joining%20your%20team%20in%20this%20section.%0A%0AJoin%20us!%0A).